### ISSUE TITLE
Composer intall, as described in documentation, points to different github repository(!)

### ISSUE THREAD

**Issue by Jacco-V** [bcit-ci/CodeIgniter4#726]
The CodeIgniter4 [Installation Documentation](https://bcit-ci.github.io/CodeIgniter4/installation/index.html#composer-installation) says:

> ### Composer Installation
> While not required, CodeIgniter can be installed via composer create-project command.
> ```
> composer create-project codeigniter4/framework
> ```

However, the packagist.org information for [`codeigniter4/framework`](https://packagist.org/packages/codeigniter4/framework)<br>
points to [**sabri-elgueder**/CodeIgniter4](https://github.com/sabri-elgueder/CodeIgniter4),  
instead of [**bcit-ci**/CodeIgniter4](https://github.com/bcit-ci/CodeIgniter4).

In other words: by following the instructions in the official documentation, you will install an outdated 'fork' of CodeIgniter4.

I can hardly believe this is as intended?

**Comment by lonnieezell** [comment]
> I can hardly believe this is as intended?

Correct. Not intended. Thanks for bringing it to our attention. 

Also - the normal warnings apply:  this is pre-alpha software so things may be broken and docs may not be up to date. But PRs are welcome.

**Comment by Jacco-V** [comment]
I'm aware of the pre-alpha state of the project and I would be happy to create a PR to mediate the issue, but the issue at hand cannot be solved, without intervention from the folks over at [packagist.org](https://packagist.org/).

Attempting to create a package named `codeigniter4/framework`, results in the following error:
![capture](https://user-images.githubusercontent.com/19333579/30716885-a2fbb1b6-9f1b-11e7-975b-b8ab1154e6a6.PNG)

**Comment by lonnieezell** [comment]
Oh, you know what - I think @jim-parry has already tried getting this handled, but I don't recall the exact details of the situation. Hopefully @jim-parry can remind us what's going on here.

**Comment by jim-parry** [comment]
Hmmm - I thought I had this handled. Will check when I get home from school!

**Comment by jaccoTopVintage** [comment]
Any updates on this issue?

**Comment by jim-parry** [comment]
This fell off my radar, sorry. Will check later today or tomorrow!

**Comment by jaynarayan89** [comment]
@jim-parry  Any update?

**Comment by jim-parry** [comment]
I have reached out to packagist.org and explained the situation, but have not heard back from them yet.

**Comment by jim-parry** [comment]
codeigniter4/framework is now ours, and bound to this repo. It should start updating shortly, at which point this issue can be closed :)

**Comment by lonnieezell** [comment]
Yay!

**Comment by jim-parry** [comment]
Verified: codeigniter4/framework now points to our repo :)

**Comment by aisfrond** [comment]
Could not find package codeigniter4/framework with stability stable.

**Comment by lonnieezell** [comment]
That's because there is no stable version of the framework. It's still in active development prior to release.
